lib/rig/distributed_set.ex

```diff
@@ -258,11 +258,9 @@ defmodule RIG.DistributedSet do
 
     n_deleted = :ets.select_delete(ets_table, match_spec)
 
-    Logger.debug(fn ->
-      if n_deleted > 0,
-        do: "Removed #{n_deleted} expired records",
-        else: :skip
-    end)
+    if n_deleted > 0 do
+      Logger.debug(fn -> "Removed #{n_deleted} expired records" end)
+    end
```

Comment:
hm easier to read but runtime performance "hit". Probably neglectable though..

Reply:
This was kind of forced with the new Elixir version, otherwise it was crashing with something like "No clause match for skip". Have better idea how to make it work?

Reply:
hm checked the implementation of `Logger` - `:skip` is not supported anymore since 1.10.